src/inputs/Table/components/TableToolbar/TableToolbar.js

```diff
@@ -4,15 +4,23 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import { useStyles } from './style';
-import { FullscreenButton } from '../FullscreenButton';
+import { FullscreenButton, ImportButton, ExportButton } from '..';
 
 export const TableToolbar = (props) => {
-  const { isFullscreen, toggleFullscreen } = props;
+  const { isFullscreen, toggleFullscreen, isReady, onImport, onExport, editMode, customToolbarActions, labels } = props;
   const classes = useStyles();
 
   return (
-    <div className={classes.tableToolbar}>
-      <FullscreenButton isFullscreen={isFullscreen} toggleFullscreen={toggleFullscreen} />
+    <div className={classes.tableToolbar} style={isReady ? { borderBottom: 'none' } : null}>
+      <FullscreenButton
+        isFullscreen={isFullscreen}
+        toggleFullscreen={toggleFullscreen}
+        disabled={!isReady}
+        label={labels.fullscreen}
+      />
+      {onImport ? <ImportButton onImport={onImport} disabled={!editMode} label={labels.import} /> : null}
+      {onExport ? <ExportButton onExport={onExport} label={labels.export} /> : null}
```

Comment:
Same remark, you must provide default values as fallback for `labels.import` and `labels.export`

Reply:
DONE